src/logger/logger.cpp

```diff
@@ -48,21 +41,22 @@ QObject* Logger::qmlSingletonRegister(QQmlEngine* engine, QJSEngine* scriptEngin
     return self();
 }
 
-void Logger::logMessage(const QString& msg, QtMsgType type)
+void Logger::logMessage(const QString& msg, const QtMsgType& type, const QMessageLogContext& context)
 {
     const QString time = QTime::currentTime().toString(QStringLiteral("[hh:mm:ss:zzz]"));
 
     // Save the message into the file
     _fileStream << QString("%1 %2\n").arg(time, msg);
 
-    const int line = _logModel.rowCount();
     // Debug, Warning, Critical, Fatal, Info
     static const QColor colors[] = { QColor("DarkGray"), QColor("orange"), QColor("red"), QColor("red"), QColor("LimeGreen") };
 
+    const int line = _logModel.rowCount();
```

Comment:
why this move

Reply:
just to be closer to the usage of the variable.